chaoslib/openebs/cstor_pool_network_delay.yml

```diff
@@ -0,0 +1,77 @@
+- name: Setup pumba chaos infrastructure
+  shell: >
+    kubectl apply -f /chaoslib/pumba/pumba_kube.yaml
+    -n {{ ns }}
+  args:
+    executable: /bin/bash
+  register: result 
+
+- name: Confirm that the pumba ds is running on all nodes
+  shell: >
+    kubectl get pod -l app=pumba 
+    --no-headers -o custom-columns=:status.phase
+    -n {{ ns }} | sort | uniq
+  args: 
+    executable: /bin/bash
+  register: result
+  until: "result.stdout == 'Running'"
+  delay: 20
+  retries: 15  
+
+- name: Getting the Application pod name
+  shell:  kubectl get pods -n {{ ns }} -l {{ app_label }} --no-headers -o=custom-columns=NAME:".metadata.name"  
+  register: pod_name
+
+- name: Getting the node on which Application pod is scheduled
+  shell: kubectl get pod {{ pod_name.stdout }} -n {{ ns }} -o jsonpath='{.spec.nodeName}'
+  register: node
+
+- name: Getting pool-pod on derived node
+  shell: kubectl get pods -n {{ target_ns }} -l app=cstor-pool -o jsonpath='{.items[?(@.spec.nodeName=="{{ node.stdout }}")].metadata.name}'  
```

Comment:
there can be multiple pools on the same node if there are many storage pool claims, and here, we may get wrong result.
You can follow the way it is in this PR - https://github.com/openebs/litmus/pull/372/files
(or) you can create a util to find the pool pod using the commands in above given PR